config/app.php

```diff
@@ -198,7 +198,6 @@
         MattAllan\LaravelCodeStyle\ServiceProvider::class,
     ],
     'local_providers' => [
-        MattAllan\LaravelCodeStyle\ServiceProvider::class,
```

Comment:
Can this now be removed from composer.json as well?

Or do we want to configure it to be "auto-discovered" as described here https://github.com/barryvdh/laravel-ide-helper#installation?

Reply:
I pushed a commit to remove it - I don't know if this was really used by anyone